scripts/test_with_authdb.sh

```diff
@@ -17,5 +17,7 @@ fi
 
 sleep 2
 
+if [ -z "$NODE_ENV" ]; then export NODE_ENV=test; fi;
```

Comment:
Tiny nit, but lets be liberal with newline characters in these scripts, we have plenty spare.

Reply:
Done.